src/Microsoft.Framework.Runtime.Hosting/project.json

```diff
@@ -4,6 +4,7 @@
     "dependencies": {
         "Newtonsoft.Json": "6.0.6",
         "NuGet.ProjectModel": "3.0.0-*",
+        "NuGet.RuntimeModel": "3.0.0-*",
```

Comment:
The project parser needs it because it has to load the `runtimes` section as a graph. In theory the Runtime doesn't need to care about that but detaching that will take a bit.